src/App.tsx

```diff
@@ -15,37 +17,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  sort_field_alphabet = 'alphabet',
+  sort_field_length = 'length',
+  sort_field_none = '',
```

Comment:
```suggestion
  Alphabet = 'alphabet',
  Length = 'length',
  None = '',
```
use PascalCase for enum keys. Also, no need to duplicate SortField in each key - it's already clear from the enum name